recipes/haven/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "haven" %}
```

Comment:
This overlaps with other packages on `pypi`. I'm not sure how best to deal with this.

Reply:
Yeah, I've also noticed that. Perhaps augment it with the organization name, such as `jonmorton-haven`?

Reply:
Renamed to `jonmorton-haven`